doc/userguide/rules/app-layer.rst

```diff
@@ -8,18 +8,31 @@ Match on the detected app-layer protocol.
 
 Syntax::
 
-    app-layer-protocol:[!]<protocol>;
+    app-layer-protocol:[!]<protocol>(,<mode>);
 
 Examples::
 
     app-layer-protocol:ssh;
     app-layer-protocol:!tls;
     app-layer-protocol:failed;
+    app-layer-protocol:!http,final;
+    app-layer-protocol:http,toserver; app-layer-protocol:tls,toclient;
```

Comment:
Might be nit, but... why use `toserver` and `toclient` here, if in other rule options we have `to_server` and `to_client`? wouldn't it be better to keep this standardized?

Reply:
Sounds good